nilearn/_utils/fixes/sklearn_basic_backports.py

```diff
@@ -0,0 +1,103 @@
+"""
+Back-porting some fundamental sklearn API.
+
+"""
+# Author: DOHMATOB Elvis
+
+import numbers
+import numpy as np
+import scipy.sparse as sp
+from sklearn.utils import as_float_array
+from sklearn.preprocessing import LabelBinarizer as SklLabelBinarizer
+
+
+def center_data(X, y, fit_intercept, normalize=False, copy=True,
+                sample_weight=None):
+    """
+    Centers data to have mean zero along axis 0. This is here because
+    nearly all linear models will want their data to be centered.
+
+    If sample_weight is not None, then the weighted mean of X and y
+    is zero, and not the mean itself
```

Comment:
you also seem to divide by the variance -> 0-mean-centering + unit-variance scaling